### PR TITLE
convert issue templates to forms

### PR DESCRIPTION
**Pull request by MattDHill**
No idea how to preview these forms without merging to master.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature-request.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+name: 💡 Feature Request
+description: Suggest an idea for EmbassyOS
+title: '[feat]: '
+labels: ["Bug", "Needs Triage"]
```
Comment (kn0wmad) [inline .github/ISSUE_TEMPLATE/feature-request.yml:4]: These labels wouldn't be appropriate for a feature request.  "Enhancement" is the only one that comes to mind

File: .github/ISSUE_TEMPLATE/feature-request.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+body:
+  - type: checkboxes
+    attributes:
+      label: Prerequisites
+      description: Please confirm you have completed the following.
+      options:
+        - label: I have searched for [existing issues](https://github.com/start9labs/embassy-os/issues) that already report this problem, without success.
```
Comment (kn0wmad) [inline .github/ISSUE_TEMPLATE/feature-request.yml:11]: This also may need rewording to reflect that they have searched for the feature request already.  There is also a section for this in our Discussions, I believe

**Review by kn0wmad — COMMENTED** [review]
The detailed info is extremely valuable to us.  My only comment is that the more complex the form, the more likely that someone may choose not to fill it out.  I don't think that's a reason to simplify forms, just an observation.

**Comment by kn0wmad** [comment]
> No idea how to preview these forms without merging to master.

Chris and I had the same realization.  Seems like a pretty glaring UX feature to miss

**Review by kn0wmad — DISMISSED** [review]
Looks good

**Review by ProofOfKeags — APPROVED** [review]

**Pull request merged**